app/mesh/changelog.md

```diff
@@ -6,18 +6,29 @@ no_version: true
 
 ## 1.6.0
 
-> Released on TBA
+> Released on 2022/02/24
 
 ### Changes
 
-Built on top of [Kuma <version>](changelog link)
+Built on top of [Kuma 1.5.0](https://github.com/kumahq/kuma/releases/tag/1.5.0)
 
-- features TBA
+- UBI images support.
+- ECS EC2 and Fargate first party support.
+- Update OPA agent to v0.37.2.
```

Comment:
If possible, these entries should include more info and links to docs (or at least links to docs, if time is the concern)